src/lib/logger.ts

```diff
@@ -345,16 +345,23 @@ export class Logger {
                 : LogEventName.COLLAPSE_COMMENT_THREAD_FOR_DOCUMENT
     };
     const event = eventMap[action];
-
+    let tileType = undefined;
     if (isSectionPath(focusDocumentId)) {
-      Logger.logCurriculumEvent(event, focusDocumentId, { tileId: focusTileId, commentText });
+      if (focusTileId) {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const [unit, facet, investigation, problem, section] = parseSectionPath(focusDocumentId) || [];
```

Comment:
Simplify to
```suggestion
        const [facet, section] = parseSectionPath(focusDocumentId) || [];
```
and while it's not technically part of this PR, I would do the same in lines 268-270 where this code was presumably copied from.

Reply:
`parseSectionPath` returns an array not an object so I think that's why it is used this way.

Reply:
You're right -- to eliminate the `eslint-disable` it would need to be:
```suggestion
        const [ , facet, ,  , section] = parseSectionPath(focusDocumentId) || [];
```

Reply:
Something like [this](https://eslint.org/docs/latest/rules/no-unused-vars#destructuredarrayignorepattern) could also be used to eliminate the warning, but that's a topic for another day. 😉